sys/src/libthread/BUILD

```diff
@@ -8,7 +8,7 @@ cc_library(
             "//amd64/include",
         ],
 	srcs = [
-		"amd64.c",
+		"$ARCH.c",
```

Comment:
I'd suggest making it always work one way. We can see that went really well with Go, people always know there's one and only one way to do things.